spec/controllers/collection_items_controller_spec.rb

```diff
@@ -15,7 +29,63 @@
       end
     end
 
-    context "create" do
+    context "rejected parameter for collection with items in" do
+      let(:owner) { @collection.owners.first.user }
+
+      it "includes rejected items" do
+        fake_login_known_user(owner)
+        get :index, collection_id: @collection.name, rejected: true
+        expect(response).to have_http_status(:success)
+        expect(assigns(:collection_items)).to include @rejected_work_item
+      end
+
+      it "excludes approved and invited items" do
+        fake_login_known_user(owner)
+        get :index, collection_id: @collection.name, rejected: true
+        expect(assigns(:collection_items)).not_to include @approved_work_item
+        expect(assigns(:collection_items)).not_to include @invited_work_item
+      end
+    end
+
+    context "invited parameter for collection with items in" do
+      let(:owner) { @collection.owners.first.user }
+
+      it "includes invited items" do
+        fake_login_known_user(owner)
+        get :index, collection_id: @collection.name, invited: true
+        expect(response).to have_http_status(:success)
+        expect(assigns(:collection_items)).to include @invited_work_item
+      end
+
+      it "excludes approved and rejected items" do
+        fake_login_known_user(owner)
+        get :index, collection_id: @collection.name, invited: true
+        expect(assigns(:collection_items)).not_to include @approved_work_item
+        expect(assigns(:collection_items)).not_to include @rejected_work_item
+      end
+    end
+
+    context "for collection with items in" do
```

Comment:
This should probably mention we're testing the approved page, since the previous context mentions we're testing the rejected page.